### DISCUSSION TITLE
How to use a company's internal npm package in Sandpack?

### DISCUSSION THREAD

**Discussion by PeanutQAQ** [codesandbox/sandpack#863] [Q&A]
1. First, I have read the content in https://sandpack.codesandbox.io/docs/guides/private-packages, but I understand it as a complete solution for a private registry.
2.Now, the situation I am facing is that my company has its own npm registry, and I am wondering how I can use the packages from the company's npm registry. 
<img width="833" alt="截屏2023-03-30 20 08 37" src="https://user-images.githubusercontent.com/25059001/228831174-d816517c-a247-4a5c-aba1-a59b7d64a933.png">

I am new to this and would appreciate your help.

**Comment by PeanutQAQ** [comment]
<img width="971" alt="截屏2023-03-30 20 35 12" src="https://user-images.githubusercontent.com/25059001/228837676-2d1e2b4e-34d6-4f98-88ca-9635c951f1df.png">
by the way, Official Website this img is not loaded, so is there any info that i lost?

**Comment by danilowoz** [comment]
Hey, thanks for opening this issue. 
I didn't really get what's the problem you're facing. Could you please give a us a example or evidence of any error you're facing?

**Reply by PeanutQAQ** [reply]
Sincerely thank you for your reply, the problem I encountered is the same as this issue（https://github.com/codesandbox/sandpack/issues/895）, could you please check the details of this issue and give me guidance to solve the problem. Once again, thank you for your time and response!
